app/models/akka/responder/GroupCallRegistry.scala

```diff
@@ -24,29 +24,57 @@ class GroupCallRecord {
 /**
  * Manages the call bindings for multi-recipient responses.
  */
-abstract class GroupCallRegistry(log: LoggingAdapter, ownId: String) {
+abstract class GroupCallRegistry(persistenceBehavior: PartOfPersistenceBehavior) {
 
-  private var bindings = Map.empty[Int, GroupCallRecord]
+  private var _bindings = Map.empty[Int, GroupCallRecord]
```

Comment:
Doesn't really matter, but i think we can skip _ in ptivate fields.

Reply:
IMHO it's a good idea since we use "_bindings_ as a backing field

Reply:
ok